lib/tesla/middleware/meta_logger.ex

```diff
@@ -24,6 +25,12 @@ if Code.ensure_loaded?(Tesla) do
     the values will be replaced with `[FILTERED]`, defaults to: `[]`.
     * `:filter_query_params` - The query params that should not be logged,
     the values will be replaced with `[FILTERED]`, defaults to: `[]`
+    * `:filter_body` - The request and response body patterns that should not be logged,
```

Comment:
Please add this information also to https://github.com/FindHotel/meta_logger/tree/29a568ad9902d8d693ec73cc7f0edb4e3476a669#options.